File: frontend/frontend-customer/src/app/models/Email.ts
export class Email{

    name?:String 
    emailAdress?: String
    subject?: String
    body?: String

    
    constructor({name, emailAdress, subject, body}:{name?: String, emailAdress?: String, subject?: String,  body?: String}) {
        this.name = name
        this.emailAdress = emailAdress
        this.subject = subject
        this.body = body
    }

} 

File: backend/Core/src/CoreEntities/src/Models/Email.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CoreEntities.src.Models
{
    public class Email
    {
        public string? Name { get; set; } = string.Empty;
        public string? EmailAdress { get; set; } = string.Empty;
        public string? Subject { get; set; } = string.Empty;
        public string? Body { get; set; } = string.Empty;

    }
}